### ISSUE TITLE
Tags allowing create work - not instant and slow

### ISSUE THREAD

**Issue by strets123** [thesgc/chembiohub_helpdesk#672]
The current implementation of tags allowing create and choice allowing create is very slow to load the project list because it loads all possibilities only once. We need to integrate with elasticsearch like on the search page. Also need to make this work for assayreg.

**Comment by aekhendry** [comment]
This functionality has been achieved, however some additional small bugs have been discovered and added to the backlog by Andy. As such marking this issue as tested successfully. chrome tagnew

**Comment by strets123** [comment]
Choice allowing create does not provide any of the choices, only allows you to create.
Tags when saved look like this in the table : ,Adam,Another
Is it right to be able to put in two or more of the same tag?
Does entering a number after a word in text search break it? e.g. ‘Choice 3’
What happened to the highlight function while searching? I liked it. 
Edit on tags allowing create – firstly, none of the currently registered tags are showing. Is this intentional? Secondly, newly entered tags disappear on adding a new one. If I click save then they appear in the table, so it is working. However I believe you should see the new tag during the edit process so the user knows it has been registered by the system.
[11:07:45] Adam Hendry: Would you like me to add them to seperate cards? i appreciate the highlight and text search are seperate issues

**Comment by strets123** [comment]
All of the above should now be fixed both on hub and tagnew

**Comment by aekhendry** [comment]
It's VERY picky, but could we have a space after each comma in the table when multiple tags are shown.

It's still not quite right:

Choice with create is not intuitive. In edit mode to enter a new term you must: type new entry, Press enter. Click the word you just typed in the drop down. Then it is saved. It should be a case of type word,  click enter and its in.

Tags allowing create is not allowing me to create new tags in edit mode.

tested on hub chrome

**Comment by aekhendry** [comment]
Someting else I've just found - In the handontable in edit mode, i can change the choice only value to anthing i want

**Comment by aekhendry** [comment]
Most of the functionality has been fixed on this card, anything remaining has been put on a new card (704/705) for the sake of clarity. As such I'm marking as done. hub chrome
